feat(curriculum): add animations to block 3.2 of the English Curriculum (#54033)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657e77d93552a0c1b00f775a.md

dashedName: task-31
---

<!-- (Audio) Linda: Would you like to go cycling with me one of these days? -->

# --description--

> The phrase `Would you like to` is often used when proposing or suggesting an activity. When Linda says `Would you like to go cycling with me one of these days?`, it is pretty much like saying `Hey, let's go cycling together.`

Reviewer: ```suggestion
The phrase `Would you like to` is often used when proposing or suggesting an activity. When Linda says `Would you like to go cycling with me one of these days?`, it's similar to saying `Hey, let's go cycling together.`
```
